plugins/lookup/onepassword.py

```diff
@@ -582,6 +580,10 @@ def get_raw(self, item_id, vault=None, token=None):
 
         return self._run(args)
 
+    def get_raw(self, item_id, vault=None, token=None):
+        args = ["item", "get", item_id, "--format", "json"]
+        return self._add_parameters_and_run(args, vault=vault, token=token)
```

Comment:
Another approach could be moving this to a `_run()` method on the `OnePassCLIv2` class.